tests/test_pipeline/test_pipeline.py

```diff
@@ -1221,6 +1222,15 @@ def test_predict_return_components(
                 "model.depth": IntUniformDistribution(low=1, high=11, step=1),
                 "model.l2_leaf_reg": LogUniformDistribution(low=0.1, high=200.0),
                 "model.random_strength": LogUniformDistribution(low=1e-05, high=10.0),
+                "transforms.0.day_number_in_week": CategoricalDistribution([False, True]),
```

Comment:
Why don't you add the same test for ensembles and hierarchical pipeline?

Reply:
Because for now ensembles doesn't work with `params_to_tune`, they fail with `NotImplementedError`.

Reply:
As for `HierarchicalPipeline`, I forgot about it. I'll add the test.